css_def/def.go

```diff
@@ -0,0 +1,48 @@
+package css_def
```

Comment:
Removed the formatter related code, kept the `css_def.DefData` type, because it's used within `graph.go` 
addressed on: https://github.com/sourcegraph/srclib-css/pull/2/commits/359f6786cf14509fe399259379992bad5a8f93b8